.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -11,5 +11,5 @@
      url: https://forums.swift.org/tags/c/development/tooling/39/swift-syntax
      about: Ask and answer questions about SwiftSyntax
    - name: SwiftSyntax Documentation
-     url: https://github.com/apple/swift-syntax/tree/main/Documentation
+     url: https://swiftpackageindex.com/apple/swift-syntax/documentation/swiftsyntax
```

Comment:
Didn't you already change this in another PR?

Reply:
Oh, rebase mistake